.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
What is the purpose of this file?  I'm not sure why we need it.

Reply:
first time I see it :)

Reply:
Right, let's remove it then. :)